README.md

```diff
@@ -498,6 +498,10 @@ Verifies the simplest way of doing authn/authz.
 Authentication is HTTP `Basic`, with users/passwords/roles defined in `application.properties`.
 Authorization is based on roles, restrictions are defined using common annotations (`@RolesAllowed` etc.).
 
+### `security/formAuthN`
+
+Verifies form-based authentication. 
```

Comment:
Also verifies that Basic Authentication is not used as a fallback option when it is explicitly disabled.